django_netjsongraph/base/link.py

```diff
@@ -39,6 +39,7 @@ class AbstractLink(TimeStampedEditableModel):
                            blank=True,
                            load_kwargs={'object_pairs_hook': OrderedDict},
                            dump_kwargs={'indent': 4})
+    last_status_change = models.DateTimeField(auto_now=True)
```

Comment:
Renamed to `status_changed` for consistency with the `created` and modified `fields`